workers/nlp/annotate/tasks.py

```diff
@@ -3,37 +3,53 @@
 import logging
 
 from utils.celery_client import celery_app
-from workers.base_task import FileTask
+from workers.base_task import ObjectTask
 
 from workers.nlp.annotate.annotate import annotate
 
 log = logging.getLogger(__name__)
 
 
-class AnnotateTask(FileTask):
+class AnnotateTask(ObjectTask):
     """
     Annotates a given text.
 
-    Annotates the text in nlp_text.txt with the params in
-    nlp_params.json and stores the annotations in annotations.json.
+    Annotates the text and stores the annotations in the json.
     """
     name = "nlp.annotate"
 
-    def process_file(self, file, target_dir):
-        new_file = self.generate_filename(file, 'json')
-        new_file_path = os.path.join(target_dir, new_file)
+    def process_object(self, obj):
+        txt_folder = os.path.join(obj.path, "data", "txt")
```

Comment:
Ich fände es besser, wenn der txt-Folder hier nicht hardcoded wäre sondern eine Representation als Parameter an den Task übergeben würde.

Es ist schon absehbar, dass wir die Repräsentationen in Zukunft nicht nur nach Dateiformaten benennen. Dann müsste man es nur in der chain ändern und nicht im Code.

Reply:
Gut. Hast du ein bespiel, wie ich das bekomme? Irgendwas in Richtung
    ```obj.get_representation_dir(Object.INITIAL_REPRESENTATION)```
richtig? Aber was gebe ich da als Argument an, um (momentan) data/txt zu bekommen?

Reply:
so in etwa dachte ich mir das:
```python
obj.get_representation_dir(self.get_param('representation'))
```